### PR TITLE
Webcam command

### PR DESCRIPTION
**Pull request by JurisMajors**
can record or take pictures :)
example usage:
``` * webcam capture 10 ```
will record a ~10 video from webcam.

### CODE REVIEW

File: b0tweb/OpenTK.dll.config [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  <dllmap os="osx" dll="libGLESv2.dll" target="/System/Library/Frameworks/OpenGLES.framework/OpenGLES" />
+  <dllmap os="osx" dll="opencl.dll" target="/System/Library/Frameworks/OpenCL.framework/OpenCL"/>
+  <dllmap os="osx" dll="SDL2.dll" target="libSDL2.dylib"/>
+  <!-- XQuartz compatibility (X11 on Mac) -->
+  <dllmap os="osx" dll="libGL.so.1" target="/usr/X11/lib/libGL.dylib"/>
+  <dllmap os="osx" dll="libX11" target="/usr/X11/lib/libX11.dylib"/>
+  <dllmap os="osx" dll="libXcursor.so.1" target="/usr/X11/lib/libXcursor.dylib"/>
+  <dllmap os="osx" dll="libXi" target="/usr/X11/lib/libXi.dylib"/>
+  <dllmap os="osx" dll="libXinerama" target="/usr/X11/lib/libXinerama.dylib"/>
+  <dllmap os="osx" dll="libXrandr.so.2" target="/usr/X11/lib/libXrandr.dylib"/>
```
Comment (x0rz3q) [inline b0tweb/OpenTK.dll.config]: Why do we need osx dll maps

Comment (JurisMajors) [reply b0tweb/OpenTK.dll.config]: the emgucv library spawned this config so .. not sure. we can remove this.

Comment (x0rz3q) [reply b0tweb/OpenTK.dll.config]: Could you remove it, and then test if everything still works 😄 ?

Comment (JurisMajors) [reply b0tweb/OpenTK.dll.config]: I removed it, seems to work. ;p :D

Comment (x0rz3q) [reply b0tweb/OpenTK.dll.config]: Damn, you threw away the entire thing. I thought about only throwing away the "osx" stuff :p

**Pull request merged**